src/pages/home/sidebar/OptionRow.js

```diff
@@ -49,13 +54,14 @@ const OptionRow = ({
     onSelectRow,
     onAddToGroup,
     isChatSwitcher,
+    isOnRightDockedModal,
     showSelectedState,
     isSelected,
 }) => {
     const textStyle = optionIsFocused
         ? styles.sidebarLinkActiveText
         : styles.sidebarLinkText;
-    const textUnreadStyle = option.isUnread
+    const textUnreadStyle = (option.isUnread || isOnRightDockedModal)
```

Comment:
This maybe should be `showOptionsAsUnread` with a default value of `true`

Reply:
Or maybe just a prop to `showAdditionalOptionStates` which would cover unread, draft comment, pinned etc

Reply:
I think it makes more sense when we can set the prop to true on calling it.
In this case, instead of naming it `showAdditionalOptionStates`, we name it `hideAdditionalOptionStates`, and default it to false.

Reply:
Done!